packages/ap-ui/docs/templatetags.rst

```diff
@@ -539,6 +545,18 @@ Add kwargs to a :tfilter:`url_with_perm` or :tfilter:`url` filter.
 
     {% component "a" href="my_url_name"|url_with_perm|with_kwargs:my_kwargs %}Link{% endcomponent %}
 
+.. templatefilter:: with_params
+
+``with_params``
+--------------------
+
+Add query params to a :tfilter:`url_with_perm` or :tfilter:`url` filter.
+
+.. code-block:: html+django
+
+    {% query_params x=1 id=record.pk as my_query %}
+    {% Button href="my_url"|url_with_perm|with_query:my_query %}
+
```

Comment:
could you expand docs to include all usages (e.g. passing a string directly or a dict from a template var)

Reply:
can you also update url and url_with_perm to mention how you can add query params